readalongs/api.py

```diff
@@ -259,3 +265,47 @@ def convert_to_readalong(
     ).decode("utf8")
 
     return xml_text + "\n"
+
+
+def convert_to_offline_html(
```

Comment:
suggest renaming "convert_tokens_to_offline_html" since it's ambiguous what is being converted from the fn name

Reply:
I get the point, but I actually disagree: "convert tokens" simplifies too much and deviates more than I'm confortable with from what this function actually converts, which is "a list of sentences/paragraphs/pages of tokens". I'd rather leave it out of the function name and have the docstring and argument list clarify.

I'd be happier with `convert_prealigned_text_to_offline_html` if we want a better function name, and then I would also change the other function to `convert_prealigned_text_to_readalong`.

Reply:
done